addons/mail/static/src/web/activity/activity.js

```diff
@@ -51,6 +51,57 @@ export class Activity extends Component {
         this.attachmentUploader = useAttachmentUploader(this.thread);
     }
 
+    get activityInfo() {
+        return markup(
+            sprintf(
+                _t(
+                    `<span class="fw-bolder %(classes)s }}">%(delay)s:</span> <span class="fw-bolder">%(activity name)s</span> <span class="o-mail-Activity-user">for %(responsible employee)s</span>`
+                ),
```

Comment:
The html should be wrapped inside the getter and not exposed as a translation key.